packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -111,6 +111,11 @@ _Props/Attrs_
     [isEmpty](https://tiptap.dev/api/editor#is-empty) API. The component and the Angular directive will have a getter method
     that can be used to bind it in the Angular application.
 -   `fitToContent` - is a boolean attribute allows the text area to expand vertically to fit the content.
+-   `maxlength` - is a number attribute that restricts the number of input characters in the editor. This is achieved through
+    Tiptap's [CharacterCount extension](https://tiptap.dev/api/extensions/character-count#limit). However, we will not display
+    the number of characters currently entered in the editor along with the maximum length (e.g., 10/100 characters entered).
+    We use this extension only to limit the number of characters that can be entered into the editor.
+-   `disabled` - is a boolean attribute to disable the editor by restricting all the user interactions within the component.
```

Comment:
Does the disabled state apply to content that the client slotted in too?

Reply:
For slotted buttons that has not been the pattern used by other controls. i.e. action buttons in a text field need to be disabled by the host application. I'd expect stuff placed in the footer slot to have state managed by the app and nimble will control layout / placement.